workloadinterface/workloadmethods.go

```diff
@@ -691,3 +699,80 @@ func (w *Workload) GetPodStatus() (*corev1.PodStatus, error) {
 	}
 	return &status, nil
 }
+
+// GetVolumeMountsPaths returns all volume mounts paths of the workload
+// if readOnly is nil, returns all volume mounts paths
+// if readOnly is true, returns only readOnly volume mounts paths
+// if readOnly is false, returns only readWrite volume mounts paths
+// if mustHaveHostVolume is true, returns only volume mounts paths that have host volume
+func (w *Workload) GetVolumeMountsPaths(readOnly *bool, mustHaveHostVolume bool) ([]VolumeMount, error) {
```

Comment:
This function should not be in this package, it is designed for a specific use case, so implement it where you need this logic...